packages/react-sdk/src/core/components/CallLayout/LivestreamLayout.tsx

```diff
@@ -102,19 +102,21 @@ export const LivestreamLayout = (props: LivestreamLayoutProps) => {
           muteAudio // audio is rendered by ParticipantsAudio
         />
       )}
-      <ParticipantView
-        className={clsx(
-          hasOngoingScreenShare &&
-            'str-video__livestream-layout__floating-participant',
-          (hasOngoingScreenShare &&
-            floatingParticipantProps?.position &&
-            `str-video__livestream-layout__floating-participant--${floatingParticipantProps.position}`) ??
-            'str-video__livestream-layout__floating-participant--top-right',
-        )}
-        participant={currentSpeaker}
-        ParticipantViewUI={FloatingParticipantOverlay || Overlay}
-        muteAudio // audio is rendered by ParticipantsAudio
-      />
+      {currentSpeaker && (
+        <ParticipantView
+          className={clsx(
+            hasOngoingScreenShare &&
+              'str-video__livestream-layout__floating-participant',
+            (hasOngoingScreenShare &&
+              floatingParticipantProps?.position &&
+              `str-video__livestream-layout__floating-participant--${floatingParticipantProps.position}`) ??
+              'str-video__livestream-layout__floating-participant--top-right',
```

Comment:
```suggestion
              `str-video__livestream-layout__floating-participant--${floatingParticipantProps.position ?? 'top-right'}`),
```
Shorter and still readable I think. 🤔

Reply:
(consider it a nitpick) 😄

Reply:
good point. I just updated your suggestion as well:
- before `floatingParticipantProps.position ?? 'top-right'`
- after `floatingParticipantProps?.position ?? 'top-right'`